bin/load-env.sh

```diff
@@ -11,10 +11,18 @@ then
   exit 1
 fi
 
-if [ ! -f "$FLUO_PROPS" ]
-then
+if [ -z "$APP_NAME" ]; then
+  echo "APP_NAME is not set!" 
+  exit 1
+fi
+
+if [ ! -d $FLUO_HOME/apps/$APP_NAME ]; then
+  $FLUO_HOME/bin/fluo new $APP_NAME
```

Comment:
+1 after `fluo new` is moved to configure script.  Would not want `fluo new` to run if I ran `print.sh` w/o configuring.